frontend/src/features/idir/search/components/PermitResendDialog.tsx

```diff
@@ -27,38 +42,74 @@ export default function PermitResendDialog({
   permitId,
   permitNumber,
   email,
-  fax,
 }: Readonly<{
   shouldOpen: boolean;
   onResend: (
     permitId: string,
     email: string,
-    fax?: Nullable<string>,
+    notificationTypes: EmailNotificationType[],
   ) => Promise<void>;
   onCancel: () => void;
   permitId: string;
   permitNumber: string;
   email?: string;
-  fax?: string;
 }>) {
+  const [notificationTypes, setNotificationTypes] = useState([
```

Comment:
I think we can simply use an object instead of an array and convert it to an array at the time of making the API call. Makes state handling a lot simpler.
```
{
    EMAIL_PERMIT: false,
    EMAIL_RECEIPT: false
}
```
This would remove the need for `isNotificationTypeSelected` function for instance as you can simply use `notificationTypes.EMAIL_PERMIT`